scheduler/src/test/scala/com/sky/kms/e2e/package.scala

```diff
@@ -8,7 +8,7 @@ import scala.concurrent.duration._
 
 package object e2e {
 
-  val Tolerance = 200 millis
+  val Tolerance = 400 millis
```

Comment:
With the assertion enabled, I found the test to be flaky with values under 400 millis